.gitignore

```diff
@@ -14,4 +14,5 @@ docs
 node_modules/
 *.log
 *~
+.idea
```

Comment:
what's this?

Reply:
Hey Chris, this is just some misc files generated by my IDE